atlasdb-dbkvs-tests/src/test/java/com/palantir/atlasdb/keyvalue/dbkvs/DbkvsPostgresTestSuite.java

```diff
@@ -79,6 +79,7 @@ public static DbKeyValueServiceConfig getKvsConfig() {
                 .dbPassword(ImmutableMaskedValue.of("palantir"))
                 .host(postgresAddress.getHostName())
                 .port(postgresAddress.getPort())
+                .maxConnections(1)
```

Comment:
As discussed in person, would be good to parameterize this test for the default and 1